wls-monitoring-service/src/test/java/de/muenchen/oss/wahllokalsystem/monitoringservice/configuration/UserInfoAuthoritiesServiceTest.java

```diff
@@ -56,7 +56,7 @@ public RestTemplate build() {
     class LoadAuthorities {
 
         @Test
-        void buildAuthoritiesFromTemplateResponseWithCollection() {
+        void should_returnVersion_when_setInApiDoc() {
```

Comment:
der Name passt nicht

Reply:
in [9fdf6aa](https://github.com/it-at-m/Wahllokalsystem/pull/578/commits/9fdf6aa9ce335d5a6cbf6e049920ea8156742c2d) gefixed